pkgs/servers/zigbee2mqtt/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix-update
+#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix-update wget
```

Comment:
We might as well rewrite this to use curl, which is already in there.

Reply:
Switched to `curl`.